documentation/tutorial/01_maps_and_tiles.md

```diff
@@ -0,0 +1,157 @@
+## Tile Class
+
+So let's start with the basics. The first thing you are going to need when building
+your game is a map. In Goby, a map is composed of several Tiles (instances of the
+Tile class), so it is important to first understand how a Tile works.
+
+When initializing a new Tile, it is possible to simply call `Tile.new` without any
+arguments. Doing so will create a instance of Tile that has the following basic
+default attributes:
+
+- It will be passable (@passable === true)
+- It won't be seen (@seen === false)
+- It will have a blank description (@description === "")
+- It will have no events (@events === [])
+- It will have no monsters (@monsters === [])
+- It will use the default graphic (@graphic === ".")
+
+Many of these attributes mean nothing now, and we will get to them, but let's focus
+on the simplest way in which we can extend this class. The defaults of the Tile class
+can be thought of as representing unremarkable blank space on your map. But what if
+we wanted to create a grassy meadow? That is where extending the `Goby::Tile` class
+comes in. Here is a small example:
+
+```
+class Grass < Goby::Tile
+  def initialize(passable: true, seen: false, description: "", events: [], monsters: [])
+    @passable = passable
```

Comment:
You can use `super`(which calls the `Tile` constructor) to simplify the `Grass` constructor.

Something like 

```ruby
super(passable: passable, seen: seen, ..., graphic: default_graphic)
```